src/server/scripts/World/go_scripts.cpp

```diff
@@ -1212,7 +1212,56 @@ class go_bells : public GameObjectScript
         return new go_bellsAI(go);
     }
 };
+/*####
+## go_ETC70_music
+####*/
+
+enum ETC70Music 
+{
+    POWER_OF_THE_HORDE = 181075
+};
```

Comment:
```suggestion
enum ETC70Music
{
    POWER_OF_THE_HORDE = 181075
};
```
My fault, stupid code suggestion added a whitespace here that needs to be removed